packages/sites/src/share-items-to-site-groups.ts

```diff
@@ -46,3 +46,7 @@ export function shareItemsToSiteGroups(
     )
   );
 }
+
+// Temporary to avoid a complex multi-project update
```

Comment:
please just add `DEPRECATED` here so it will come up next time we search the code for deprecated things